### DISCUSSION TITLE
How to install tailwindcss version 2

### DISCUSSION THREAD

**Discussion by Alamnzr123** [tailwindlabs/tailwindcss#8053] [Help] [answered]
Hello, i have project that require for tailwind version 2, do you know code for it

thanks
![Screenshot (2282)](https://user-images.githubusercontent.com/54619997/161925121-c1c5d4ea-ef0f-42c4-a74b-4e08dcfbcadf.png)

**Comment by NovusOpus** [comment, marked as answer]
Like this: 
```console
npm i -D tailwindcss@2.2.19
```

**Comment by AviAvinav** [comment]
I would like to extend to @NovusOpus's answer. 

The structure of the config file in Tailwind CSS v2 was different from v3, so you should check out this link: https://v2.tailwindcss.com/docs/installation for more info on it depending on your framework or build tools.

Also, as mentioned in @NovusOpus's answer do `npm i -D tailwindcss@2.2.19` because in the above link it says to do `npm i -D tailwindcss` which is wrong as it will install Tailwind CSS v3.
